actors/builtin/verifreg/verified_registry_actor.go

```diff
@@ -50,10 +50,17 @@ type AddVerifierParams struct {
 }
 
 func (a Actor) AddVerifier(rt vmr.Runtime, params *AddVerifierParams) *adt.EmptyValue {
+	if params.Allowance.LessThan(MinVerifiedDealSize) {
+		rt.Abortf(exitcode.ErrIllegalArgument, "Allowance %d below MinVerifiedDealSize for add verifier %v", params.Allowance, params.Address)
+	}
+
 	var st State
 	rt.State().Readonly(&st)
 	rt.ValidateImmediateCallerIs(st.RootKey)
 
+	if params.Address == st.RootKey {
```

Comment:
Please TODO and reference https://github.com/filecoin-project/specs-actors/issues/556 here and anywhere a user-provided address is inspected. Until we do that, this isn't really enforced.

Reply:
This is done for all methods that need this check.

Reply:
This is done everywhere.